core/src/main/java/io/grpc/internal/Stream.java

```diff
@@ -79,6 +79,9 @@
    */
   boolean isReady();
 
+
+  void start();
```

Comment:
`newStream()` shouldn't really be doing anything. Any validation could happen in `start()`. What sort of errors are you imagining?